src/spectro_inlets_quantification/sensitivity.py

```diff
@@ -1137,3 +1140,59 @@ def plot_F_vs_f(
             F_fit = self.alpha * f_fit
             ax.plot(f_fit, F_fit, "k--")  # type: ignore
         return ax
+
+
+def append_sensitivity_factors(*args):
+    """Recursively append elements in a list"""
+    sf = append_two_sensitivity_factors(args[0], args[1])
+    if len(args) == 2:
+        return sf
+    remaining_args = [sf] + list(args[2:])
+    return append_sensitivity_factors(*remaining_args)
+
+
+def append_two_sensitivity_factors(
+        sf_1: Union[SensitivityFactor, SensitivityList],
+        sf_2: Union[SensitivityFactor, SensitivityList]
+) -> SensitivityList:
+    """This should be incorporated into __add__ of SensitivityFactor and SensitivityList
```

Comment:
So, while I appreciate that this may be more that you bargained for in this PR, I think it would be nice if this code was in fact merged into those classes. The thing is that the whole outcome tree below is fairly complex. Merging it into the classes would eliminate some of that complexity because one of the types would be known.